lib/instances/BaseInstance.lua

```diff
@@ -180,6 +180,27 @@ function BaseInstance.prototype:GetChildren()
 	return result
 end
 
+-- GetDescendants implementation using a queue rather than recursion
```

Comment:
I don't think this remark is very important, unless you also mention _why_ we'd opt for a non-recursive solution!